projects/Rockchip/devices/RK3568/README.md

```diff
@@ -0,0 +1,30 @@
+# RK3568
```

Comment:
I'd rather call the device `RK356X` as it's more clear and reflects the actual SoCs which are supported here. There's no reason to do so, since you introduced this extra `uboot_helper`  `soc` for distinction anyway. From what I can see the only place that needed a change is `projects/Rockchip/bootloader/release` for copying the device trees: I would just always replace `X` in `${DEVICE}` with an empty string - this would also be benifical for other SoCs (e.g. RK322X, if it ever makes it upstream LE  :laughing: )

Reply:
I agree and have now renamed the device to `RK356X` and changed to remove `X` from `DEVICE` when copying `.dtb`-files. A rebuild seem to generate a similar image.